src/api/tax-free-asset/content-types/tax-free-asset/schema.json

```diff
@@ -0,0 +1,34 @@
+{
+  "kind": "collectionType",
+  "collectionName": "tax_free_assets",
+  "info": {
+    "singularName": "tax-free-asset",
+    "pluralName": "tax-free-assets",
+    "displayName": "Tax Free Assets",
+    "description": ""
+  },
+  "options": {
+    "draftAndPublish": false
+  },
+  "pluginOptions": {},
+  "attributes": {
+    "chainId": {
+      "type": "enumeration",
+      "enum": [
+        "MAINNET",
+        "GNOSIS_CHAIN",
+        "ARBITRUM_ONE",
+        "BASE",
+        "SEPOLIA"
```

Comment:
You should make it a FK to the Network model 
<img width="755" alt="image" src="https://github.com/user-attachments/assets/225cea39-c6a7-4f1d-a3bf-ccb312016790" />

Reply:
Thank you! Fixed

Reply:
Have you pushed it already?